guacamole-common-js/src/main/webapp/modules/StringWriter.js

```diff
@@ -161,7 +161,12 @@ Guacamole.StringWriter = function(stream) {
 
         // Fill buffer with UTF-8
         for (var i=0; i<text.length; i++) {
-            var codepoint = text.charCodeAt(i);
+            var codepoint = text.codePointAt(i);
```

Comment:
Need to add the check for `codePointAt()`, here, as well.

Reply:
Done